Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -7,9 +10,15 @@ public static class ArrayOperations
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
+        /// 
+        private static bool IsArrayNullOrEmpty(int[] array)
+        {
+            return (array == null || array.Length == 0);
+        }
+
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (!IsArrayNullOrEmpty(array)) Array.Sort(array);
```

Comment:
I kind of assumed that people would manually implement it. So please do a mnual implementation of a sort 😄